### PR TITLE
feat: add `p/moul/{md,debug,web25}` + update `r/moul/home`

### PR DESCRIPTION
**Pull request by moul**
Related with https://hackmd.io/a8k09_TeQUu6WawvkpDDpw?both

- [x] `p/moul/web25` - displays a link suggesting to view the realm from an external webui, but avoid recursive printing when seen from the external webui
- [x] `p/moul/md` - minimal `markdown` helpers library
- [x] `p/moul/debug` - displays useful informations when adding `?debug=1`
- [x] update `r/moul/home`
	- [x] markdown table with links example
	- [x] svg example
	- [x] use of the new `p/moul/...` packages
- [x] Release a static web25 interface somewhere (https://github.com/moul/gno-moul-home-web25)

### CODE REVIEW

**Comment by Gno2D2** [comment]
I'm a bot that assists the Gno Core team in maintaining this repository. My role is to ensure that contributors understand and follow our guidelines, helping to streamline the development process.

The following requirements must be fulfilled before a pull request can be merged.
Some requirement checks are automated and can be verified by the CI, while others need manual verification by a staff member.

These requirements are defined in this [configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

## Automated Checks

 🟢 Maintainers must be able to edit this pull request ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
 🔴 The pull request head branch must be up-to-date with its base ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch))


## Manual Checks

*No manual checks match this pull request.*

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>Maintainers must be able to edit this pull request (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Maintainer can modify this pull request

```
</blockquote></details>

<details><summary>The pull request head branch must be up-to-date with its base (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🔴 Requirement not satisfied
└── 🔴 Head branch (moul:dev/moul/md-workflows) is up to date with base (master): behind by 188 / ahead by 2

```
</blockquote></details>

</blockquote></details>



</blockquote></details>

File: examples/gno.land/r/moul/config/config_test.gno
Comment (leohhhn) [inline examples/gno.land/r/moul/config/config_test.gno:1]: Leftover file?

**Review by leohhhn — APPROVED** [review]

File: examples/gno.land/r/moul/config/config_test.gno
Comment (moul) [reply examples/gno.land/r/moul/config/config_test.gno:1]: It's to ensure that `gno test` does not skip the folder. Although there are no unit tests, it still passes through the gnovm syntax checks and other validations.

**Pull request merged**
